app/src/main/java/org/fossasia/openevent/general/event/EventIdConverter.kt

```diff
@@ -5,12 +5,14 @@ import android.arch.persistence.room.TypeConverter
 class EventIdConverter {
 
     @TypeConverter
-    fun fromEventId(eventId: EventId): Long{
-        return eventId.id
+    fun fromEventId(eventId: EventId?): Long?{
+        return eventId?.id
     }
 
     @TypeConverter
-    fun toEventId(id: Long): EventId{
-        return EventId(id)
+    fun toEventId(id: Long?): EventId?{
+        return id?.let {
+            EventId(it)
+        }
```

Comment:
@dreadpool2 is this needed?

Reply:
App was crashing as eventId parameter was null for some events. I will check again :+1:

Reply:
No

Reply:
Then this should remain as it is ?

Reply:
Yes. There are other things wrong. I'll review soon

Reply:
Updated the logic.

Reply:
@iamareebjamal Could you please review this?